k8s/examples/wordpress/mysql/00-service-account.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: wordpress-mysql
+  namespace: default
```

Comment:
Please don't hard code the namespace. It is intended for people to be able to apply these yamls and provision the solution in the namespace they prefer.

Same with the rest of files that mention the ns.